controllers/getCategories.controller.js

```diff
@@ -42,3 +43,15 @@ exports.getReviewComments = (req, res, next) => {
       next(err);
     });
 };
+exports.postReviewComments = (req, res, next) => {
+  const { review_id } = req.params;
+  const newComment = req.body;
+
+  postComment(review_id, newComment)
+    .then((comments) => {
+      res.status(201).send({ comments });
```

Comment:
I'm being slightly picky, but maybe change this to a key of 'comment' rather than 'comments', as it's just a single comment being returned. 🙂